### PR TITLE
Added Homepage for ecommerce

### PR DESCRIPTION
**Pull request by Hamza-777**
Added homepage for my ecommerce website please review and provide feedback.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+    color: var(--white-color)
+}
+
+.sports-wear {
+    background: url('./resources/sportswear.jpg') center;
+    color: var(--white-color)
+}
+
+.games {
+    background: url('./resources/gaming.jpg') center;
```
Comment (vrx29) [inline style.css:283]: If these are product images, then try to add them using img src="" tag, not like this.

Comment (Hamza-777) [reply style.css:283]: No it is meant to be for styling. You can check the live page. And then give me a feedback.

Comment (vrx29) [reply style.css:283]: ok fine then . No issues

**Review by vrx29 — APPROVED** [review]

**Review by animeshgarai09 — COMMENTED** [review]
try to use hex code for colors

**Comment by venkateshwarans** [comment]
> try to use hex code for colors

May I know why not rgba?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+}
+
+body {
+    background: var(--light-color);
+}
+
+img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
```
Comment (venkateshwarans) [inline style.css:38]: Try to play with `object-position` and `object-fit: cover` for the homepage main slider.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+    background: url('./resources/mobilecovers.jpg') center;
+    color: var(--white-color)
+}
+
+.merch {
+    background: url('./resources/merchandise.jpg') center;
+    color: var(--white-color)
+}
+
+.btn-to {
```
Comment (venkateshwarans) [inline style.css:302]: what does `btn-to` mean? `to` abbreviates something?

**Review by venkateshwarans — APPROVED** [review]
Overall looks good. Good work. Just added a couple of comments.

File: style.css (hunk 2 above)
Comment (Hamza-777) [reply style.css:38]: Okay nice I was thinking what to do to make it cover the whole width and look good at the same time.

File: style.css (hunk 3 above)
Comment (Hamza-777) [reply style.css:302]: Yeah, I named it to buttons that take you to that related product listing page.

**Comment by Hamza-777** [comment]
> Overall looks good. Good work. Just added a couple of comments.

Thanks bhai.

**Comment by Hamza-777** [comment]
> try to use hex code for colors

Where in the code can you point out?

**Pull request merged**